lib/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/receive_enter_data_state.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_state.dart';
+import 'package:seeds/v2/datasource/remote/model/balance_model.dart';
+import 'package:seeds/v2/domain-shared/page_command.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+class NavigateToReceiveDetails extends PageCommand {}
+
+class ShowTransactionFailSnackBar extends PageCommand {}
+
+/// --- STATE
+class ReceiveEnterDataState extends Equatable {
+  final PageState pageState;
+  final PageCommand? pageCommand;
+  final RatesState ratesState;
+  final bool isAutoFocus;
```

Comment:
is this needed? Now that we have a special widget for the amount entry.

Reply:
Change to hardcoded bool true, It does not affect this screen, If it those in the future when more logic is added, will add a pr to modified 'Entry amount widget'